### ISSUE TITLE
iemukern.kext在哪啊，我没找到

### ISSUE THREAD

**Issue by 234321121** [iqemu64/iqemu64#4]
sudo cp -r ${HOME}/Develop/kernels/iemukern.kext /tmp/
sudo kextutil -t /tmp/iemukern.kext
xcrun simctl install <device> <pathOf.appFolder>
我的qq是234321121，请大佬指点，iemukern.kext在哪啊，我没找到，不知道device填什么

**Comment by flyqie** [comment]
先执行setup.sh完成编译

device写你要安装app的模拟器的device_id，用xcrun simctl list获取

**Comment by goodtomeetyou** [comment]
> 先执行setup.sh完成编译
> 
> device写你要安装app的模拟器的device_id，用xcrun simctl list获取

Did you ran the project successfully? Does it still work?

**Comment by flyqie** [comment]
> > 先执行setup.sh完成编译
> > device写你要安装app的模拟器的device_id，用xcrun simctl list获取
> 
> Did you ran the project successfully? Does it still work?

Unfortunately, I ended up running into some runtime issues, so I wasn't able to get it to work.

**Comment by goodtomeetyou** [comment]
> Unfortunately, I ended up running into some runtime issues, so I wasn't able to get it to work.

I understand, this is the second emulator I found that anybody had success to run it. Updating the patches for iOS/macOS 26 is getting harder than I thought.

**Comment by flyqie** [comment]
> > 先执行setup.sh完成编译
> > device写你要安装app的模拟器的device_id，用xcrun simctl list获取
> 
> Did you ran the project successfully? Does it still work?

I did not continue to study this project, but I can be sure that the technical route of the project is feasible, because the "Heilei Simulator" software also uses this type of technical route.

**Comment by goodtomeetyou** [comment]
> I did not continue to study this project, but I can be sure that the technical route of the project is feasible, because the "Heilei Simulator" software also uses this type of technical route.

Yeah, I actually thought they was using this codebase for their product. I'm pretty sure this project is still useful for it's purpose. I was trying to contact the devs with no luck so I could keep working on it.

**Comment by flyqie** [comment]
> > I did not continue to study this project, but I can be sure that the technical route of the project is feasible, because the "Heilei Simulator" software also uses this type of technical route.
> 
> Yeah, I actually thought they was using this codebase for their product. I'm pretty sure this project is still useful for it's purpose. I was trying to contact the devs with no luck so I could keep working on it.

In fact, the technical route of this project is similar to Intel Bridge Technoly, but the difficulty is that we need to deal with the closed-source iOS and its severely castrated Xcode simulator. So I think this technical route is feasible but difficult (especially when it needs to be compatible with new systems).

Good luck and hope you succeed!

**Comment by goodtomeetyou** [comment]
> So I think this technical route is feasible but difficult (especially when it needs to be compatible with new systems).
> 
> Good luck and hope you succeed!

Yes, those static assembly patches we're designed to be not scalable and currently it requires a ton of reverse egineering to update everything. Not sure if it's worth my time haha. Thank you!
